srcs/main.c

```diff
@@ -11,7 +11,7 @@ int	main(int ac, char **av)
 	cub = init_cub();
 	validate_map(map_fd, av[1], &cub);
 	print_cub(&cub);
-	init_mlx(&cub);
+	init_mlx_and_raycast(&cub);
```

Comment:
I think maybe it would be best to group everything above raycast inside a separate intializations function - as these are basically all set up so the raycasting and hooking can do their magic.

Reply:
Ya, I agree